edge/pkg/eventbus/mqtt/server.go

```diff
@@ -107,10 +108,13 @@ func (m *Server) onSubscribe(msg *packet.Message) {
 	// for "SYS/dis/upload_records", no need to base64 topic
 	var target string
 	resource := base64.URLEncoding.EncodeToString([]byte(msg.Topic))
-	if strings.HasPrefix(msg.Topic, "$hw/events/device") || strings.HasPrefix(msg.Topic, "$hw/events/node") {
+	if strings.HasPrefix(msg.Topic, dtcommon.DeviceETPrefix) || strings.HasPrefix(msg.Topic, dtcommon.MemETPrefix) {
```

Comment:
One mudule shouldn't depend on another, we need to move these const to a common place.